packages/flutter/test/material/chip_theme_test.dart

```diff
@@ -105,6 +117,7 @@ void main() {
       brightness: Brightness.dark,
       elevation: 5,
       pressElevation: 6,
+      iconTheme: customIconTheme,
```

Comment:
Here we'll need to increment the previous color  (`checkmarkColor: Color(0xfffffff7)` 
So this becomes:
`iconTheme: IconThemeData(color: Color(0xfffffff8)),`

This will expect a hash code when matching `IconThemeData`. We need slightly updated the test to use `equalsIgnoringHashCodes` on the list of strings.

```dart
    expect(
      description,
      equalsIgnoringHashCodes(<String>[
        'backgroundColor: Color(0xfffffff0)',
        'deleteIconColor: Color(0xfffffff1)',
        'disabledColor: Color(0xfffffff2)',
        'selectedColor: Color(0xfffffff3)',
        'secondarySelectedColor: Color(0xfffffff4)',
        'shadowColor: Color(0xfffffff5)',
        'surfaceTintColor: Color(0xfffffff8)',
        'selectedShadowColor: Color(0xfffffff6)',
        'showCheckmark: true',
        'checkMarkColor: Color(0xfffffff7)',
        'labelPadding: EdgeInsets.all(1.0)',
        'padding: EdgeInsets.all(2.0)',
        'side: BorderSide(width: 10.0)',
        'shape: RoundedRectangleBorder(BorderSide(width: 0.0, style: none), BorderRadius.zero)',
        'labelStyle: TextStyle(inherit: true, size: 10.0)',
        'secondaryLabelStyle: TextStyle(inherit: true, size: 20.0)',
        'brightness: dark',
        'elevation: 5.0',
        'pressElevation: 6.0',
        'iconTheme: IconThemeData#00000(color: Color(0xfffffff8))',
      ]),
    );
```

Reply:
@timmaffett Please add `equalsIgnoringHashCodes` from above.

The test is failing because of this. This ignores the hashcode generated by `IconThemeData`.